### PR TITLE
!

### PR DESCRIPTION
**Pull request by wigzard**
Look

### CODE REVIEW

File: src/test/java/dalTest/loadDataFromDBTest/LoadDataActorsTest.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import com.story.dataAccessLayer.dataActions.RetrieveActorAction;
+import com.story.dataAccessLayer.dataDescriptors.ActorDescriptor;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by User on 22.08.2016.
+ */
+//This TestClass designed to check the correctness of the RetriveActorAction class  with the database
```
Comment (Alex9876) [inline src/test/java/dalTest/loadDataFromDBTest/LoadDataActorsTest.java]: All comments should be in 
/**
* Your comment
**/

File: src/test/java/dalTest/loadDataFromDBTest/LoadDataActorsTest.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
+package dalTest.loadDataFromDBTest;
+
+import com.story.dataAccessLayer.dataActions.RetrieveActorAction;
+import com.story.dataAccessLayer.dataDescriptors.ActorDescriptor;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by User on 22.08.2016.
```
Comment (Alex9876) [inline src/test/java/dalTest/loadDataFromDBTest/LoadDataActorsTest.java]: Please change your name. 'User' it's not so good

File: src/test/java/dalTest/loadDataFromDBTest/LoadDataActorsTest.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+ */
+//This TestClass designed to check the correctness of the RetriveActorAction class  with the database
+public class LoadDataActorsTest {
+
+    RetrieveActorAction retrieveActorAction; //This veriable using for conection to database
```
Comment (Alex9876) [inline src/test/java/dalTest/loadDataFromDBTest/LoadDataActorsTest.java]: All comments should be in 
/**
* Your comment
**/

**Comment by wigzard** [comment]
I changed comment

**Pull request merged**